src/Components/Publishing/Display/DisplayPanel.tsx

```diff
@@ -260,6 +263,44 @@ export class DisplayPanel extends Component<Props, State> {
     return isVideo
   }
 
+  toggleMuted = (e) => {
+    e.stopPropagation()
+    this.setState({ isMuted: !this.state.isMuted })
+  }
+
+  renderVideoIcon() {
+    const { isPlaying, isMuted } = this.state
+    const { isMobile } = this.props
+
+    if (isMobile) {
+      return null
+    } else if (isPlaying && isMuted) {
+      return (
+        <MutedIcon onClick={this.toggleMuted}>
+          <svg width="30" height="30" xmlns="http://www.w3.org/2000/svg">
+            <g fill="#FFF">
+              <path
+                d="M29.514 28L19.13 17.614 9.557 8.043 1.514 0 0 1.514l6.529 6.529H.757V21.47H11.83l7.3 7.286v-8.114L28 29.514zM19.129.757l-7.3 7.286 7.3 7.3z"
+              />
+            </g>
+          </svg>
+        </MutedIcon>
+      )
+    } else if (isPlaying && !isMuted) {
+      return (
+        <UnmutedIcon onClick={this.toggleMuted}>
+          <svg width="30" height="30" xmlns="http://www.w3.org/2000/svg">
+            <g fill="#FFF">
+              <path d="M18 12.864V0l-7.152 7.286H0v13.428h10.848L18 28V12.864z" />
+            </g>
+          </svg>
+        </UnmutedIcon>
```

Comment:
Not necessary now, but we have an icons dir: https://github.com/artsy/reaction/blob/master/src/Components/Publishing/Icon/IconEditEmbed.tsx